octopusCats/src/main/scala/octopus/cats/package.scala

```diff
@@ -0,0 +1,23 @@
+package octopus
+
+import _root_.cats.data.{Validated, ValidatedNel, NonEmptyList}
+
```

Comment:
I would prefer to `import cats.syntax.validated._` and then `value.valid` or `value.invalid`